projects/ngx-valdemort/src/lib/validation-errors.component.html

```diff
@@ -1,5 +1,8 @@
 <ng-container *ngIf="shouldDisplayErrors">
   <div [class]="errorClasses" *ngFor="let errorDirective of errorDirectivesToDisplay">
-    <ng-container *ngTemplateOutlet="errorDirective.templateRef; context: {$implicit: label, error: actualControl.errors[errorDirective.type]}"></ng-container>
+    <ng-container *ngTemplateOutlet="errorDirective!.templateRef; context: {
+      $implicit: label,
+      error: actualControl.errors![errorDirective.type]
```

Comment:
@jnizet I went the easy way and just assumed `actualControl.errors` is not null, but I guess that's always true?